wiki/Multiplayer/en.md

```diff
@@ -1,451 +1,419 @@
+[Multi_mods_player image]: ./img/Multi_mods_player.jpg "Player (osu!) mods options (With Free Mods on)"
+[Multi_beatmapinfo image]: ./img/Multi_beatmapinfo.jpg "Beatmap information when hovered over."
+[Multi_host_song_change image]: ./img/Multi_host_song_change.jpg "When the Host is changing songs, this bar will become a temporary placeholder."
+[Multi_SS image]: ./img/Multi_SS.jpg "Example of how the Song Selection will look like for host."
+[Multi_mh_single image]: ./img/Multi_mh_single.jpg "Example of Match History."
+[Multi_mh_team image]: ./img/Multi_mh_team.jpg "Example of Team Match History."
+
+<!-- Team Mode images -->
+[HTH Interface image]: ./img/Multi_HTH.jpg "Head-to-Head Interface"
+[Team Vs Interface image]: ./img/Multi_team_vs.jpg "Team Vs Interface"
+[Tag Coop Interface image]: ./img/Multi_tag_co-op.jpg "Tag Coop Interface"
+[Tag Team Vs Interface image]: ./img/Multi_tag_team_vs.jpg "Tag Team Vs Interface"
+[HTH Grade image]: ./img/Multi_grade.jpg "Head-to-Head grade screen"
+[Team Vs Grade image]: ./img/Multi_team_grade.jpg "Team Vs grade screen"
+[Tag Coop Grade image]: ./img/Multi_co-op_grade.jpg "Team Co-Op grade screen"
+[Tag Team Vs Grade image]: ./img/Multi_tag_team_grade.jpg "Tag Team Vs grade screen"
 
 <!--Title-->
 # Multiplayer
-![Flowchart of a multi game][Multi_flowchart image]
 
 ## Introduction
-Multiplayer was implemented on the 3rd of June 2008 (3.6.08) under [osu! Public Release b335][Release b335], and is a game mode in which up to sixteen players can compete against each other on a single map decided by the host.
 
-[osu!academy][osu!academy wikilink] covered this page in [Episode 6 (6:52)][osu!academy Ep6] along with [Online Users Panel][Chat_Console#Extended_Chat_Console wikilink].
+Multiplayer or _Multi_ was implemented on the 3rd of June 2008 (3.6.08) under [osu! Public Release b335][Release b335], and is a game mode in which up to sixteen players can compete against each other on a single map decided by the host.
+
+[osu!academy][osu!academy wikilink] covered this page in [Episode 6 (6:52)][osu!academy Ep6] along with [Online Users Panel][Chat_Console#Extended_Chat Console wikilink].
 
 ## How to enter
+
 ### Minimum requirements
+
+![One of the ways to transfer the beatmap without leaving the room][Beatmaps_peppy image]
+
+_One of the ways to transfer the beatmap without leaving the room_
+
 - Decent online connection.
-- osu!account logged into Bancho (should be done automatically)
-- Have a fair amount of beatmaps in your Songs folder (unless you prefer to download as you go)
-- Knowing [how to transfer beatmaps without leaving the room.][Beatmaps2 wikilink] [Picture example here.][Beatmaps_peppy image]
-- Logical thinking (knowing when to ask about something you don't understand)
+- _osu!account_ logged into _Bancho_ (should be done automatically during ingame login)
+- Have a fair amount of beatmaps in the `Songs` folder (except for personal preference to download on the spot)
```

Comment:
Removing the line.

Not a hard requirement to _must_ have beatmaps in the Songs folder (it does speed up the player preparation(at least, for you) of the play and do not need to wait for one new fellow to get the beatmap downloaded, but that's all).